packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -164,6 +164,12 @@ enum ListTileControlAffinity {
 /// and to ensure that [subtitle] doesn't wrap (if [isThreeLine] is false) or
 /// wraps to two lines (if it is true).
 ///
+/// Leading and trailing widgets will be constrained in height according to
```

Comment:
The heights of the [leading] and [trailing] widgets are constrained according to ..